README.md

```diff
@@ -37,13 +39,13 @@ From the CDN:
 Using [npm](https://npmjs.org):
 
 ```sh
-npm install @auth0/auth0-spa-js
+npm install @auth0/auth0-spa-js@first-availability
```

Comment:
Curious if we should publish using the `fa` tag instead of `first-availability`, with flutter we also used `fa`, probably easier for the user. 

```suggestion
npm install @auth0/auth0-spa-js@fa
```